third_party/spdk_defs/nvme_defs.h

```diff
@@ -239,6 +239,128 @@ struct SglDescriptor {
 } ABSL_ATTRIBUTE_PACKED;
 static_assert(sizeof(SglDescriptor) == 16);
 
+// NVMe Base Specification Figure 245
+// https://nvmexpress.org/wp-content/uploads/NVM-Express-1_4-2019.06.10-Ratified.pdf
+struct IdentifyNamespace {
+  uint64_t nsze;  // namespace size
```

Comment:
nit: master has thus far stuck to the standard that all eligible fields are bit fields

Reply:
thanks for the reminder!